README.md

```diff
@@ -7,36 +7,72 @@
 
 ## Description
 
-A flexible front end to plate bases pipelines in Sequencescape
+A flexible front end to plate bases pipelines in Sequencescape.
 
 ## Initial Setup
 
-1. In Limber, connect to Sequencescape to configure required data:
-```shell
-  bundle exec rake config:generate
-```
+Steps must be followed in either the Sequencescape repository or this Limber
+repository, as indicated:
 
-2. In Sequencescape, configure all Limber required data:
-```shell
-  bundle exec rake limber:setup
-```
+1. In Limber, ensure the appropriate version of Ruby is installed. The command
+   here is for `rbenv` but you may want to use a different Ruby version manager.
+   If the command fails, you might need to check the list of available Ruby
+   versions using `rbenv install --list-all` and use the one that most closely
+   matches the contents of `.ruby-version`:
 
-3. Start Sequencescape (will start in 3000):
-```shell
-  bundle exec rails s
-```
+   ```shell
+   rbenv install
+   ```
 
-4. Start the delayed job processor
-```shell
-  bundle exec rake jobs:work
-```
+2. In Limber, make the Bundler gem install the dependencies for this project:
 
-5. In Limber, start Limber (will start in 3001):
-```shell
-  bundle exec rails s
-```
+   ```shell
+   bundle install
+   ```
+
+3. In Limber, install the yarn dependencies:
+
+   ```shell
+   yarn install
+   ```
+
+4. In Sequencescape, configure all Limber required data:
+
+   ```shell
+   bundle exec rake limber:setup
+   ```
+
+5. In SequenceScape, setup an API key for Limber via the Rails console:
+
+   ```shell
+   bundle exec rails console
+   new_app = ApiApplication.new(name: 'limber', contact: 'developers', key: 'development', privilege: 'full')
+   new_app.save(validate: false)
```

Comment:
This shouldn't be necessary, this should all be set-up automatically, either when running `bin/setup` in sequencescape, or individually as part of `rake db:seed` task (run on both `rake db:setup` and `rake db:create`)

Reply:
I had a lot of trouble running `bin/setup` in SequenceScape and I couldn't get it to run to completion.  However, I've dropped all the tables from MySQL again and now it worked and like you said the key was created for me.